src/topology/uniform_space/uniform_embedding.lean

```diff
@@ -133,7 +133,7 @@ have ∀b', (b, b') ∈ t → b' ∈ closure (e '' {a' | (a, a') ∈ s}),
 ⟨a, (nhds b).sets_of_superset (mem_nhds_left b htu) this⟩
 
 lemma uniform_embedding_subtype_emb (p : α → Prop) {e : α → β} (ue : uniform_embedding e)
-  (de : dense_embedding e) : uniform_embedding (de.subtype_emb p) :=
+  (de : dense_embedding e) : uniform_embedding (dense_embedding.subtype_emb p e) :=
```

Comment:
To define `dense_embedding.subtype_emb`, the assumption that `e` is a dense embedding is not necessary, so I removed it from this statement. This means that one can not access it any more with `de.subtype_emb p`, but instead with `dense_embedding.subtype_emb p e`. Arguably, one should also change the name and/or the location of the definition `dense_embedding.subtype_emb`, but since I did not see any use for it outside of this context I left it where it was. If you have a good idea, tell me!